presto-main/src/main/java/com/facebook/presto/operator/scalar/JsonOperators.java

```diff
@@ -343,6 +345,16 @@ public static long hashCode(@SqlType(JSON) Slice value)
         return value.hashCode();
     }
 
+    @ScalarOperator(INDETERMINATE)
+    @SqlType(BOOLEAN)
+    public static boolean indeterminate(@SqlType(JSON) Slice value, @IsNull boolean isNull)
+    {
+        if (isNull) {
+            return true;
+        }
+        return false;
```

Comment:
This may need to look inside the json structure. You should consider how the json type works and its semantics with respect to other sql types (I don't remember off the top of my head, but I can help you take a look)